config.yaml

```
@@ -316,6 +316,110 @@ languages:
       #  url: "/blog"
       #  weight: 5
 
+  ko:
+    languageName: 한국어 Korean
+    title: The Falco Project
+    contentDir: content/ko
+    params:
+      heroText:
+        title: The Falco Project
+        detail1: 리눅스 커널 시스템 호출을 추적합니다.
+        detail2: 컨테이너 메타 정보로 커널 이벤트를 보완합니다.
+        detail3: 보안 규칙에 대한 이벤트를 지정합니다.
+        detail4: 보안 사고 발생 시 알림을 지원합니다.
+      runtime:
+        title: 왜 런타임 보안인가?
+        intro: >-
+          쿠버네티스 보안을 위해 악의적인 예상치 못한 동작을 탐지 할 수 있도록 제어해야 합니다. 예제:
```

```suggestion
        intro: >-
               쿠버네티스 보안을 위해 악의적인 예상치 못한 동작을 탐지 할 수 있도록 제어해야 합니다. 예제:
```
- 원문과 동일하게 띄어쓰기 했습니다.